typescript/sdk/src/core/HyperlaneRelayer.ts

```diff
@@ -0,0 +1,221 @@
+import { TransactionReceipt } from '@ethersproject/providers';
```

Comment:
Import ethers exports though the 'front door' of `'ethers'` rather than these sub-packages. This type should be under the `providers` export.